Makefile

```diff
@@ -1,21 +1,15 @@
 PROJECT_HOME_DIR := ${CURDIR}
 
-build:
-    # Builds a TensorFlow 2 ART docker container
-    # IMPORTANT ! If you have an existing python env folder make sure to first add it to the `.dockerIgnore` \
-    to reduce the size of your the art docker image
-	docker build -t project-art-tf2 .
-
 build1:
 	# Builds a TensorFlow 1 ART docker container
     # IMPORTANT ! If you have an existing python env folder make sure to first add it to the `.dockerIgnore` \
     to reduce the size of your the art docker image
 	docker build -t project-art-tf1 .
 
-run-bash:
+
```

Comment:
Removing the `run-bash` target might affect developers who rely on this command to enter the docker container. Ensure there is an alternative or provide a rationale for its removal.